Gemfile

```diff
@@ -189,7 +189,7 @@ group :red_hat_virtualization, :manageiq_default do
 end
 
 group :qpid_proton, :optional => true do
-  gem "qpid_proton",                    "~>0.30.0",          :require => false
+  gem "qpid_proton",                    "~>0.37.0",          :require => false
```

Comment:
I don't think this is that simple, but @agrare can you weigh in here?

Reply:
We would need to update the qpid_proton rpm as well in order to do this

Reply:
thnx. will split up